docs/devGuide/settingUp.md

```diff
@@ -32,14 +32,16 @@ We recommend the **WebStorm IDE** for working with MarkBind code.
 ## Setting up the dev environment
 
 1. **Fork and clone** the MarkBind repo.
-1. **Install dependencies** by running `npm run ci:all`
-   <popover header="Why **2** `ci` commands?" content="Our components subpackage depends on `node-sass` indirectly, which depends on `fsevents`.<br><br> `fsevents` is not needed nor compatible with Windows, and `npm ci` does not respect this since it uses `package-lock.json`.">(`ciwin:all` if you are on a Windows machine)</popover>
-   in the root folder of your cloned repo.
-1. **To bind your cloned version of MarkBind to your console** (instead of the released version of MarkBind), run `npm link` in the root folder of the cloned repo.
+1. **Install dependencies** by running <popover content="Under the hood, this calls `npm ci` and `lerna bootstrap`">`npm run setup`</popover> in the root folder of your cloned repo.
```

Comment:
I had some issues due to existing `node_modules` directories in the existing packages. Do you think it's worth adding a note here for existing markbind devs to first run `lerna clean` before running `npm run setup`?

Reply:
`lerna clean` won't work unless you have it installed globally or have run `npm ci` (which `npm run setup` does) first unfortunately. I think the devguide should be written in the perspective of new developers always though. Experienced developers could always drop an issue / ping, and have a likely have a better understanding of node already 😁

Reply:
ah fair enough!